src/tests/JIT/Regression/JitBlue/Runtime_33972/Runtime_33972.cs

```diff
@@ -178,19 +178,14 @@ static Vector128<float> AdvSimd_CompareEqual_Vector128_Single_CreateZeroZeroZero
     {
         Vector128<float> result = default;
         var asVar = Vector128.Create(0f, 0f, 0f, 0f);
-        // 'asVar' should be propagated.
-        // 'AdvSimd.CompareEqual' should be hoisted out of the loops.
-        // ARM64-FULL-LINE: fcmeq {{v[0-9]+}}.4s, {{v[0-9]+}}.4s, #0.0
-        // ARM64: blt
-        // ARM64-NOT: fcmeq
+
```

Comment:
Is this test unused / uninteresting due to removing these lines? I thought that tests marked `<HasDisasmCheck>true</HasDisasmCheck>` are *only* used for disasm checking, and aren't actually run (but maybe that will change?). @TIHan @markples ?

Reply:
This was actually a test added quite a bit before `DisasmCheck` existed and is covering specific containment functionality: https://github.com/dotnet/runtime/issues/33972

The disasm checks were added later since it represents a good candidate for validating the containment functionality is actually working as expected.

In this particular case, we could probably cover it as something like:
```
// ARM64-FULL-LINE: fcmeq {{v[0-9]+}}.4s, {{v[0-9]+}}.4s, #0.0
// ARM64-NOT: blt
// ARM64-NOT: fcmeq
```

But the combination of loop unrolling + dead code optimization makes it a case that's a bit difficult to test/understand the reasoning around.

I think we're fine not having the coverage for these two subcases in particular given all the other coverage, but I'd be fine with adding the above lines if people would prefer they remain.

Reply:
`HasDisasmCheck` adds disasm checking in addition to execution.  (In fact, as it is, it requires execution because we get the disasm via normal execution plus DOTNET_JitDisasm.)

Loop logic is difficult to test in a line-based system.  The existence of `fcmeq` is probably covered reasonably by the other tests, so this one is a question of whether the trouble/brittleness is worth it to verify that intended loop transformations have occurred.  (and it sounds like the answer is 'no')

Reply:
> `HasDisasmCheck` adds disasm checking in addition to execution

That's good to hear. However, there's an unfortunate wrinkle: it appears that all HasDisasmCheck tests also must set:

```
DOTNET_TieredCompilation=0
DOTNET_JITMinOpts=0
```

so those tests don't get alternative stress mode configurations run.

(this is subject for another issue, not this PR, or course)